src/test/operators/aggregate_test.cpp

```diff
@@ -24,8 +25,9 @@
 
 namespace opossum {
 
+template <typename T>
 class OperatorsAggregateTest : public BaseTest {
- protected:
+ public:
```

Comment:
Since we changed this to a typed test, it only compiles if the `SetUpTestCase` is public. In the [docs for googletest](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md), we have found this note concerning this matter:

> NOTE: Though the above code declares SetUpTestSuite() protected, it may sometimes be necessary to declare it public, such as when using it with TEST_P.